mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -1,31 +1,387 @@
-use tracing::info;
+use std::{collections::HashMap, marker::PhantomData, str::FromStr, time::Duration};
 
-use crate::settings::ConnectionSettings;
+use anyhow::Result;
```

Comment:
let's not use anyhow in lib crates for error handling. use thiserror instead

Reply:
I though anyhow error work with thiserror mapping, I am confused by the examples.